src/Documentation/Installation/Prerequisites.md

```diff
@@ -5,13 +5,19 @@ title: Prerequisites
 
 # Prerequisites
 
-Orleans is a set of .NET libraries. In order to use Orleans, you need [.NET Framework](http://www.microsoft.com/net) 4.5.1 or higher and a copy of [Visual Studio](https://www.visualstudio.com) 2015 or higher. Note that the Express versions of Visual Studio do not support extension packages, but you can use Orleans by adding references to the NuGet packages directly.
+Orleans is a set of .NET libraries delivered via [NuGet packages](NuGets.md).
+In order to use Orleans, you need [.NET Framework](http://dot.net) 4.6.1 (since 1.5.0, 4.5.1 for prior versions) or higher and a copy of [Visual Studio](https://www.visualstudio.com) 2015 or higher.
+Note that the Express versions of Visual Studio do not support extension packages, but you can use Orleans by adding references to the NuGet packages directly.
 
-In production, Orleans requires persistent storage. The following technologies are supported (only need one of those):
+In production, Orleans requires persistent storage for reliable cluster membership.
```

Comment:
perhaps we should change the wording here so that we can have one consistent list instead of putting SF in a separate sentence.
eg "Production deployments of Orleans require reliable cluster membership. The membership system is pluggable and supported implementations are maintained for the following systems:"